convert2rhel/checks.py

```diff
@@ -75,77 +91,75 @@ def perform_pre_ponr_checks():
 
 
 def check_convert2rhel_latest():
-    """Make sure that we are running the latest version of convert2rhel"""
-    gpg_path = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
-    ssl_cert_path = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
-    repo_content = (
-        "[convert2rhel]\n"
-        "name=Convert2RHEL Repository\n"
-        "baseurl=https://cdn.redhat.com/content/public/convert2rhel/$releasever/x86_64/os/\n"
-        "gpgcheck=1\n"
-        "enabled=1\n"
-        "sslcacert=%s\n"
-        "gpgkey=file://%s\n" % (ssl_cert_path, gpg_path)
-    )
+    """Make sure that we are running the latest downstream version of convert2rhel"""
+    logger.task("Prepare: Checking if this is the latest version of convert2rhel")
+
+    if not system_info.has_internet_access:
+        logger.warning("Skipping the check because no internet connection has been detected.")
+        return
 
     cmd = [
         "repoquery",
         "--releasever=%s" % system_info.version.major,
+        "--repofrom='convert2rhel,https://cdn.redhat.com/content/public/convert2rhel/$releasever/$basearch/os/'convert2rhel",
+        "convert2rhel",
     ]
 
-    repo_dir = tempfile.mkdtemp(prefix="convert2rhel_repo.", dir=utils.TMP_DIR)
-    utils.mkdir_p(repo_dir)
-    repo_path = os.path.join(repo_dir, "convert2rhel.repo")
-    store_content_to_file(filename=repo_path, content=repo_content)
-
-    cmd.append("--setopt=reposdir=%s" % repo_dir)
-    cmd.append("convert2rhel")
-
     raw_output_convert2rhel_versions, return_code = run_subprocess(cmd, print_output=False)
 
     if return_code != 0:
         logger.warning(
-            "Couldn't check if this is the latest version of convert2rhel\n"
-            "repoquery failed %s, %s" % (return_code, raw_output_convert2rhel_versions)
+            "Couldn't check if the current installed convert2RHEL is the latest version.\n"
```

Comment:
```suggestion
            "Couldn't check if the current installed Convert2RHEL is the latest version.\n"
```

Convert2RHEL should be capitalized

Reply:
Thank you, I don't know why but recently I've been missing this! I will get this changed and pushed!